src/main/java/eu/enmeshed/model/notificationItem/PeerSharedAttributeSucceededNotificationItem.java

```diff
@@ -0,0 +1,17 @@
+package eu.enmeshed.model.notificationItem;
+
+import eu.enmeshed.model.attributes.IdentityAttribute;
+import eu.enmeshed.model.attributes.RelationshipAttribute;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@SuperBuilder
+public class PeerSharedAttributeSucceededNotificationItem extends NotificationItem {
+  private IdentityAttribute successorContent;
+  private RelationshipAttribute attribute;
```

Comment:
This should not be added as an additional property. 

As documented ([https://enmeshed.eu](https://enmeshed.eu/integrate/data-model-overview#peersharedattributesucceedednotificationitem)), the `PeerSharedAttributeSucceededNotificationItem` has only three properties: `successorContent`, `predecessorId` and `successorId`. 

The type of `successorContent` can be an IdentityAttribute or a RelationshipAttribute. The line `private RelationshipAttribute attribute;` should therefore be removed again. Instead, the type of `successorContent` should be extended so that not only IdentityAttributes but also RelationshipAttributes are permitted. 

This can be realized by changing line `private IdentityAttribute successorContent;` to `private Attribute successorContent;` (see other comment).

Reply:
Thanks! It is clear now. Changed.